app/assets/javascripts/components/tickets/sidebar.jsx

```diff
@@ -48,6 +48,11 @@ export class Sidebar extends React.Component {
               : 'Course Unknown'
           }
         </section>
+        <section className="related-tickets">
+          <Link target="_blank" to={`/tickets/dashboard?search_by_course=${ticket.project.slug}`}>
+            Search all tickets for course: {ticket.project.title}
```

Comment:
Since you've already created an i18n message for this, you may as well use it here.

Reply:
You will say that I am nitpicking :), but I did not use i18n here because there were none, so I guess I had to blend with the existing code.

Also, the i18 string I have added is : "Search all tickets for this course" and is used for the other place.
The string here is without "this".
We would have "Search all tickets for this course: course XXX".
It seems a bit of a repetition.
However, English is not my mother tongue :) and I'll make the change.

Reply:
You're right, it's a bit different. In this case, I think I'd rather have a reusable i18n that works for both places, rather than the perfect message that avoids repetition. But I think 'this course' or 'course' are not needed here anyway. One alternative would be 'Search all tickets for: XXX' (which I'm fine with not using i18n for). That would make the sidebar a little less cluttered too.